src/app.js

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+
+console.log("Script is running");
+
+
+
+async function fetchCountryData(name) {
+
+    // countryName.innerHTML = ``;
+    // errorMessage.innerHTML = ``;
+
+        try {
+            const response = await axios.get(`https://restcountries.com/v2/name/${name}`)
+            const countries = response.data;
+            console.log(countries);
+
+            const countryName = document.getElementById('country-name');
+
+            countries.map((country) => {
```

Comment:
Je zou hier nog spreading kunnen toepassen, zodat je niet overal `country.` voor hoeft te zetten
```suggestion
            countries.map(({ name, subregion, population, currencies }) => {
```